tests/Security/Http/Authentication/AuthenticationSuccessHandlerTest.php

```diff
@@ -178,6 +213,35 @@ private function getDispatcher()
         return $dispatcher;
     }
 
+    private function getFalseRememberDeviceResolver()
+    {
+        $rememberDeviceResolver = $this->createMock(RememberDeviceResolver::class);
+
+        $rememberDeviceResolver
+            ->expects($this->any())
+            ->method('getRememberDeviceStatus')
+            ->willReturn(false);
+
+        return $rememberDeviceResolver;
+    }
+
+    private function getTrueRememberDeviceResolver()
+    {
+        $rememberDeviceResolver = $this->createMock(RememberDeviceResolver::class);
+
+        $rememberDeviceResolver
+            ->expects($this->any())
+            ->method('getRememberDeviceStatus')
+            ->willReturn(true);
+
+        return $rememberDeviceResolver;
+    }
```

Comment:
you could consider making this 1 function which accepts a boolean parameter to reduce the amount of duplicate code lines..